src/deluge/model/instrument/midi_instrument.h

```diff
@@ -137,5 +137,6 @@ class MIDIInstrument final : public NonAudioInstrument {
 	void outputAllMPEValuesOnMemberChannel(int16_t const* mpeValuesToUse, int32_t outputMemberChannel);
 	Error readMIDIParamFromFile(Deserializer& reader, int32_t readAutomationUpToPos,
 	                            MIDIParamCollection* midiParamCollection, int8_t* getCC = NULL);
-	MIDILabelCollection midiLabelCollection;
+
+	deluge::fast_map<uint8_t, std::string_view> labels;
```

Comment:
Just want to confirm, does this change have any impact on the weight of the label strings in memory?

I know it was one of the reasons Mark pushed me to use a resizable array.

Reply:
No if anything, it's lighter.